libcob/call.c

```diff
@@ -106,14 +108,26 @@ lt_dlerror (void)
 /* note: only defined in configure when HAVE_DLFCN_H is true and dlopen can be linked */
 #include <dlfcn.h>
 
-#define lt_dlopen(x)	dlopen(x, RTLD_LAZY | RTLD_GLOBAL)
 #define lt_dlsym(x,y)	dlsym(x, y)
 #define lt_dlclose(x)	dlclose(x)
 #define lt_dlerror()	dlerror()
 #define	lt_dlinit()
 #define	lt_dlexit()
 #define lt_dlhandle	void *
 
+void* lt_dlopen(const char* filename) {
+	
+	if (cobsetptr == NULL || cobsetptr->cob_load_global == NULL) {
+		// TODO: What to do when cobsetptr is null?
+	}
+
```

Comment:
the setting is already available in all places we call this - so drop that check here